_posts/2019-10-18-projects.md

```diff
@@ -112,7 +112,7 @@ Our Open Life Science program is based on the [Mozilla Open Leader program](http
 
 - [Open Yeast Engineering Project (OYEP)](https://mozilla.github.io/leadership-training/round-5/projects/#open-yeast-engineering-project-oyep) - Timothy Stiles
 
-    Would you drink a GMO beer? What if you brewed it yourself? The open yeast engineering project (OYEP) at BosLab is an open-ended exploration into yeast engineering. We get together, run experiments, do analyses, and engineer yeast strains to produce flavors, fragrances, and small molecules with the hopes of dextinctifying forgotten flavors or using some resultant strains to produce glowing banana flavored chocolate stout and artificial rosé. 
+    Would you drink a GMO beer? What if you brewed it yourself? The open yeast engineering project (OYEP) at BosLab is an open-ended exploration into yeast engineering. We get together, run experiments, do analysis, and engineer yeast strains to produce flavors, fragrances, and small molecules with the hopes of dextinctifying forgotten flavors or using some resultant strains to produce glowing banana flavored chocolate stout and artificial rosé. 
```

Comment:
The word "analyses" actually works perfectly in this context...as it is referring to not just one, but multiple examinations.